package.json

```diff
@@ -14,7 +14,8 @@
     "patch-package": "^6.4.7",
     "prettier": ">=2",
     "prettier-plugin-import-sort": "^0.0.7",
-    "ts-node": "^10.8.1"
+    "ts-node": "^10.8.1",
+    "@chakra-ui/cli": "^latest" 
```

Comment:
Thanks for the fix! :+1: 
Can we set an explicit version here instead of `latest`?